lnst/Tests/LongLivedConnections.py

```diff
@@ -0,0 +1,174 @@
+from .BaseTestModule import BaseTestModule
+import socket
+import logging
+import threading
+import time
+import select
+import resource
+from typing import Union
+
+from lnst.Common.IpAddress import Ip4Address, Ip6Address, BaseIpAddress
+from lnst.Common.Parameters import IpParam, IntParam, ListParam, StrParam
+
+
+def get_inet(ip: Union[Ip4Address, Ip6Address]):
```

Comment:
I'd rename this to `get_af()`, it is address family, result can be either AF_INET or AF_INET6